gui/src/components/dashboard/dashboard.vue

```diff
@@ -16,13 +16,25 @@
 */
 <template>
```

Comment:
why do we require any change in this file

Reply:
Reverted the formatting related changes